src/main/antora/modules/security/pages/security.adoc

```diff
@@ -174,9 +182,13 @@ public class TestIdentityStore implements IdentityStore {
 }
 ----
 
-This identity store only validates the single identity (user) "john", with password "secret1" and roles "user" and "caller". Defining this kind of identity store is often the simplest way to get started. Note that Jakarta Security doesn't define a simple identity store out of the box, because there are questions about whether that would promote security best practices.
+This identity store only validates the single identity (user) "john", with password "secret1" and roles "user" and "caller". Defining this kind of identity store is often the simplest way to get started. 
+
+[NOTE]
+Jakarta Security doesn't define a simple identity store out of the box. The reason is that everything in Jakarta Security promotes best practices, and it's not clear if a simple identity store fits in with those best practices.
```

Comment:
'define' blocked my active reading, after re-reading I think 'provide' would be clearer.